experimental/unitsconversion/src/converter.rs

```diff
@@ -0,0 +1,133 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use crate::{
+    measureunit::{MeasureUnit, MeasureUnitParser},
+    provider::{MeasureUnitItem, UnitsInfoV1},
+    ConversionError,
+};
+use litemap::LiteMap;
+use zerovec::ZeroSlice;
+
+/// Represents the possible cases for the convertibility between two units.
+pub enum Convertibility {
+    /// The units are convertible.
+    /// For example, `meter` and `foot` are convertible.
+    Convertible,
+
+    /// The units are reciprocal.
+    /// For example, `meter-per-second` and `second-per-meter` are reciprocal.
+    Reciprocal,
```

Comment:
Does `Reciprocal` only apply to exact reciprocals, or anything that's convertible with an inverse?